Sources/Clappr/Classes/Helper/AnimationDuration.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+public class AnimationDuration {
```

Comment:
`ClapprAnimationDuration` would make more sense in this case (and probably avoid any collision). What do you guys think?